test/functional/3rd-party-repo/3rd-party-repo-remove-negative.bats

```diff
@@ -0,0 +1,88 @@
+#!/usr/bin/env bats
+
+# Author: Karthik Prabhu Vinod
+# Email: karthik.prabhu.vinod@intel.com
+
+load "../testlib"
+
+test_setup(){
+
+	create_test_environment "$TEST_NAME"
+
+	contents=$(cat <<- EOM
+		[test1]
+		url=www.abc.com
+		\n
+		[test2]
+		url=www.efg.com
+		\n
+		[test4]
+		url=www.pqr.com
+		\n
+		[test5]
+		url=www.lmn.com
+		\n
+	EOM
+	)
+
+	repo_config_file="$STATEDIR"/3rd_party/repo.ini
+	run sudo sh -c "mkdir -p $STATEDIR/3rd_party/"
+	run sudo sh -c "mkdir -p $PATH_PREFIX/opt/3rd-party/{test1,test2,test3,test4,test5}"
+	write_to_protected_file -a "$repo_config_file" "$contents"
```

Comment:
what is the point of adding all this to the test setup if it is not being used/validated anywhere on the tests. The tests are only checking the return code and the output, so what's the point of all these?
This gets proven as not useful since the directories created with this command `run sudo sh -c "mkdir -p $PATH_PREFIX/opt/3rd-party/{test1,test2,test3,test4,test5}"`are not correct since you are using the wrong directory (3rd-party instead of 3rd_party) and yet the test doesn't complain at all.